Kingfisher/ImageCache.swift

```diff
@@ -584,13 +581,9 @@ public extension ImageCache {
             let resourceKeys = [NSURLIsDirectoryKey, NSURLTotalFileAllocatedSizeKey]
             var diskCacheSize: UInt = 0
             
-            if let fileEnumerator = self.fileManager.enumeratorAtURL(diskCacheURL,
-                includingPropertiesForKeys: resourceKeys,
-                options: NSDirectoryEnumerationOptions.SkipsHiddenFiles,
-                errorHandler: nil) {
-                    
-                    for fileURL in fileEnumerator.allObjects as! [NSURL] {
-                        
+            if let fileEnumerator = self.fileManager.enumeratorAtURL(diskCacheURL, includingPropertiesForKeys: resourceKeys, options: NSDirectoryEnumerationOptions.SkipsHiddenFiles, errorHandler: nil),
```

Comment:
Length Violation: Line should be 100 characters or less: currently 201 characters